### DISCUSSION TITLE
Exclude tsconfig.json

### DISCUSSION THREAD

**Discussion by ghost** [swc-project/swc#1884] [Q&A]
I have the folder called: Project. In that folder I have the **.swcrc** file. Also in folder Project, there is a folder called src. In the src folder, there are 2 other folders (client & server). Both folders client and server have a tsconfig.json file, that is being compiled. How can I exclude it using the exclude tag in **.swcrc**?

I also use this in package.json: `npx swc ./src/server -d ./packages/server--copy-files`

**Comment by theLAZYmd** [comment]
Bump
